src/Cake.Svn.Tests/Unit/Add/SvnAddSettingsExtensionsTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using Cake.Svn.Add;
+using Xunit;
+
+namespace Cake.Svn.Tests.Unit.Add
+{
+    public sealed class SvnAddSettingsExtensionsTests
+    {
+        public sealed class TheConstructor
+        {
+            [Fact]
+            public void Should_Throw_If_AddSettings_Is_Null()
+            {
+                // Given
+                SvnAddSettings svnAddSettings = null;
+
+                // When
+                // Then
+                Assert.Throws<ArgumentNullException>("settings", () => svnAddSettings.ToSvnAddArgs());
```

Comment:
We should add a helper for this kind of stuff (like https://github.com/cake-contrib/Cake.Issues/blob/develop/src/Cake.Issues.Testing/ExceptionAssertExtensions.cs). But can be done in a later PR.